components/ApplicationForm.tsx

```diff
@@ -51,6 +67,19 @@ export default function ApplicationForm() {
             <hr/>
             <h2>Project Details</h2>
 
+            <label>
+                Main Focus
+                <small>
+                    In which area will your project have the most impact?
+                </small>
+                <select {...register('main_focus')} >
+                    <option value="bitcoin">Bitcoin</option>
+                    <option value="lightning">Lightning</option>
+                    <option value="nostr">nostr</option>
+                    <option value="other">Other</option>
```

Comment:
should we have an option for legal defense?

Reply:
No.

The "Legal Defense Fund" is its own fund. It is not a bucket of projects that will have its own subcommittee to evaluate project applications.